scripts/lint.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-if [[ $(flutter format --line-length=120 -n .) ]]; then
+if [[ $(flutter format -n .) ]]; then
```

Comment:
sadly pub doesn't respect our 120 preference. I personally find 80 waaay too short but there is an ongoing war on flutter